packages/commerce-sdk-react/src/hooks/ShopperLogin/helper.ts

```diff
@@ -27,14 +28,16 @@ type ShopperLoginHelpersType = typeof ShopperLoginHelpers[keyof typeof ShopperLo
  * Avaliable helpers:
  * - loginRegisteredUserB2C
  * - loginGuestUser
+ * - register
  * - logout
  */
 export function useShopperLoginHelper<Action extends ShopperLoginHelpersType>(
     action: Action
 ): UseMutationResult<
     ShopperLoginTypes.TokenResponse,
     Error,
-    void | Argument<Auth['loginRegisteredUserB2C']>
+    // TODO: what's the better way for declaring the types?
+    void | Argument<Auth['loginRegisteredUserB2C']> | Argument<Auth['register']>
```

Comment:
@wjhsf I'm going to need your help to properly type this function 🙏

Reply:
I tried poking around for a few minutes, couldn't crack it. Kinda vibing just `any` with `// TODO` right now...